operations/patch.go

```diff
@@ -99,6 +99,7 @@ func Patch() cli.Command {
 			mutuallyExclusiveArgs(false, patchDescriptionFlagName, autoDescriptionFlag),
 			mutuallyExclusiveArgs(false, preserveCommitsFlag, uncommittedChangesFlag),
 			mutuallyExclusiveArgs(false, repeatDefinitionFlag, repeatPatchIdFlag),
+			mutuallyExclusiveArgs(false, repeatPatchIdFlag, includeModulesFlag),
```

Comment:
For my understanding why do these have to be mutually exclusive?

Reply:
when we use repeat patch, we use the changes from that patch including the modules so it wouldn't make sense to include another modules!